stats/opencensus/opencensus.go

```diff
@@ -78,15 +83,89 @@ func ServerOption(to TraceOptions) grpc.ServerOption {
 }
 
 // unaryInterceptor handles per RPC context management. It also handles per RPC
-// tracing and stats.
+// tracing and stats, and records the latency for the full RPC call.
 func unaryInterceptor(ctx context.Context, method string, req, reply interface{}, cc *grpc.ClientConn, invoker grpc.UnaryInvoker, opts ...grpc.CallOption) error {
-	return invoker(ctx, method, req, reply, cc, opts...)
+	startTime := time.Now()
+	err := invoker(ctx, method, req, reply, cc, opts...)
+	callLatency := float64(time.Since(startTime)) / float64(time.Millisecond)
+
+	var st string
+	if err != nil {
+		s := status.Convert(err)
+		st = canonicalString(s.Code())
+	} else {
+		st = "OK"
+	}
```

Comment:
`st := canonicalString(status.Code(err))`

(Or inline where used.)

And below.